app/views/application/_menu.html.erb

```diff
@@ -136,6 +138,10 @@
             <%= current_user.incoming_requests.open.count %>
           </span>
         <% end %>
+
+      <%= link_to "Top up", "https://zeus.gent/contact/?username=#{current_user.name}#payment-info", target: "_blank", class: "px-3 py-2 rounded-md text-sm font-medium text-gray-300 hover:bg-gray-700 hover:text-white" %>
```

Comment:
The styling is off for these buttons: 
![image](https://user-images.githubusercontent.com/915130/194247352-d5e510af-85f5-4351-84f5-994f889a57f4.png)

They should probably get `block px-3 py-2 rounded-md text-base font-medium text-gray-300 hover:bg-gray-700 hover:text-white` as styling